web-components/src/components/chat-message/ChatMessage.ts

```diff
@@ -20,6 +20,7 @@ export class ChatMessage extends LitElement {
         <md-avatar
           type=${ifDefined(this.self ? "self" : undefined)}
           title=${this.self ? "self" : this.title}
+          label="Avatar"
```

Comment:
you cannot hard code any labels in english

Reply:
Fixed